ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -164,10 +164,7 @@ class KeymanWebViewController: UIViewController {
 
   override func viewWillAppear(_ animated: Bool) {
     super.viewWillAppear(animated)
-    if #available(iOSApplicationExtension 14.0, *) {
-      self.userContentController.removeAllScriptMessageHandlers()
-    }
-    self.userContentController.add(self, name: keymanWebViewName)
+    self.userContentController.removeAllScriptMessageHandlers()
```

Comment:
Looks like you have deleted line 170 by mistake here

Reply:
yes -- thanks for spotting that!